workflows/foobernetes.yaml

```diff
@@ -98,6 +98,7 @@ steps:
       loadBalancerIP: 10.0.0.1
       location: eu1
       replica: false
+      policy: $policies.lb_policy
```

Comment:
I suggest removing the policy tag on lines 106 and 124 now when a proper policy attribute exists.

Reply:
OK, done